internal/config/image/cuda_image.go

```diff
@@ -38,8 +38,15 @@ const (
 // a map of environment variable to values that can be used to perform lookups
 // such as requirements.
 type CUDA struct {
-	env    map[string]string
+	additionalVisibleDevicesEnvVars []string
+
+	env map[string]string
+
 	mounts []specs.Mount
+
+	acceptDeviceListAsVolumeMounts bool
+	acceptEnvvarUnprivileged       bool
+	isPrivileged                   bool
```

Comment:
```suggestion
	env map[string]string
	mounts []specs.Mount
 
	preferredVisibleDeviceEnvVars       []string
	acceptDeviceListAsVolumeMounts bool
	acceptEnvvarUnprivileged       bool
	isPrivileged                   bool
```
note that I think `preferredVisibleDevicesEnvVars` better indicates the intent.